### PR TITLE
[RELEASE] Mongodb

### LINKED ISSUE #177: [DB] Alterações para mongo

### PR DESCRIPTION
**Pull request by c4mpus**
* fix: ajust compose and env.example

* feat: insert new base workflows

* fix: ajust docker compose and dockerfile

* chore: remove db url from .env.example

Closes #177 

<details open> 
  <summary>
    <b>Feature</b>
  </summary>

AJustado schema do banco de dados e integrado com a aplicação

</details>

<details open> 
  <summary>
    <b>Bugfix</b>
  </summary>

- **Description**
  N/A

- **Cause**
  N/A

- **Solution**
N/A
</details>

<details> 
  <summary>
    <b>Changelog</b>
  </summary>
N/A
</details>

<details open> 
  <summary>
    <b>Visual evidences :framed_picture:</b>
  </summary>

image

</details>

<details open> 
  <summary>
    <b>Checklist</b>
  </summary>

- [ ] Issue linked
- [ ] Build working correctly
- [ ] Tests created
</details>

<details> 
  <summary>
    <b>Additional info</b>
  </summary>
N/A
</details>

### CODE REVIEW

File: src/teste.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { PrismaClient } from '@prisma/client';
+
+const client = new PrismaClient({
+  log: ['info', 'warn', 'query', 'error'],
+});
+
+async function main() {
+  console.log('FOI');
```
Comment (Luiginicoletti) [inline src/teste.js:8]: FOI
britney-spears-reaction-gif-by-moodman

**Review by Luiginicoletti — DISMISSED** [review]

**Review by Frompaje — COMMENTED** [review]
remove o console.log

**Comment by SamuelMoraesF** [comment]
O arquivo `src/teste.js` é necessário?

File: package.json [hunk 2]
```diff
@@ -29,7 +29,8 @@
     "docs:lint": "redocly lint --extends minimal ./openapi/main.yaml",
     "docker-dev": "docker compose --profile dev up",
     "docker-prod": "docker compose --profile prod up --build",
-    "prisma:generate": "npx prisma generate"
+    "prisma:generate": "npx prisma generate",
+    "pisma:push": "npx primsa db push"
```
Comment (Frompaje) [inline package.json:33]: npx prisma db push

**Review by Frompaje — CHANGES_REQUESTED** [review]

**Review by Frompaje — APPROVED** [review]

**Review by davidambz — APPROVED** [review]

**Review by juliaam — APPROVED** [review]

**Pull request merged**